src/renderer/services/midgard/types.ts

```diff
@@ -27,19 +27,13 @@ export type PoolAssets = Asset[]
 export type PoolAssetsRD = RD.RemoteData<Error, PoolAssets>
 export type PoolAssetsLD = LiveData<Error, PoolAssets>
 
-export type AssetDetail = {
-  asset: string
-  dateCreated: number
-  priceRune: string
+export type PoolAssetDetail = {
+  asset: Asset
+  assetPrice: BigNumber
```

Comment:
actually we have [AssetWithAmount](https://github.com/thorchain/asgardex-electron/blob/develop/src/renderer/types/asgardex.ts#L12) type. maybe there is a sense to use it ?

Reply:
Yep, type is almost similar. However, `assetPrice` means price for this asset, but not amount of it. Also we might want to add other properties to `PoolAssetDetail` later (similar to originally `MidgardPoolDetail`).